internal/server/data/identity.go

```diff
@@ -144,6 +123,8 @@ func AssignIdentityToGroups(tx WriteTxn, user *models.ProviderUser, newGroups []
 				return nil, fmt.Errorf("create group: %w", err)
 			}
 			groupID = group.ID
+			// add to the map so we can use the IDs below
+			addIDs[name] = groupID
```

Comment:
This is not used until #4039, but I didn't want to lose it, so kept it in the same commit